libminifi/include/core/state/nodes/MetricsBase.h

```diff
@@ -37,28 +37,28 @@ namespace org::apache::nifi::minifi::state::response {
 class ResponseNode : public core::Connectable, public PublishedMetricProvider {
  public:
   ResponseNode()
-      : core::Connectable("metric"),
-        is_array_(false) {
+    : core::Connectable("metric"),
```

Comment:
Why did you change these? Shouldnt it be 4 space indent from the level of the constructor (which starts at 2)?

Reply:
You are right, I thought the double indentation is only needed if it conflicts with the indentation of the implementation code under it, but the [style guide](https://google.github.io/styleguide/cppguide.html#Constructor_Initializer_Lists) insists on 4 spaces all times, updated in c71bdaa3378b7087a33b7bd5f9a6bf4c3dd8cff8